packages/vhd-lib/src/openVhd.js

```diff
@@ -0,0 +1,12 @@
+import { VhdFile, VhdDirectory } from './'
+
+export const openVhd = async function (handler, path) {
```

Comment:
You can use this simpler syntax (also it makes sure that the function is correctly named, regarding of the JS engine):

```suggestion
export async function openVhd(handler, path) {
```